typescript/packages/fi-core/src/grpc/tracer.proto

```
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package tfc.tracer;
```

Buf (1.54.0)

Files with package "tfc.tracer" must be within a directory "tfc/tracer" relative to root but were in directory "typescript/packages/fi-core/src/grpc".

(PACKAGE_DIRECTORY_MATCH)